### ISSUE TITLE
[FEATURE] add Java binary variable to mcserver _default.cfg

### ISSUE THREAD

**Issue by muhdiboy** [GameServerManagers/LinuxGSM#3493]
## User Story

As a user and system administrator, I want to add a Java binary variable so that I can define which Java should be used for each mcserver server.

## Basic info

* **Distro:** Ubuntu 18.04
* **Game:** Minecraft Server
* **Command:** mcserver.cfg
* **LinuxGSM version:** v21.1.3

## Further Information

Add a default to the mentioned variable and insert the variable into the preexecutable variable.

## Further Reading

I've applied this change in my fork of the repo (develop) for mcserver, pmcserver and wmcserver server _default.cfg's.
The changes are as following:
add this to # Predefined Parameters
`javabin="/usr/bin/java"`
and insert the defined variable in ## Game Server Directories
```
- preexecutable="java -Xmx${javaram}M -jar"
+ preexecutable="${javabin} -Xmx${javaram}M -jar"
```

**Comment by dgibbs64** [comment]
Im guessing this is due to Minecraft updating the java requirements. I assume this is to allow older builds to be able to run on the same servers as newer builds?

**Comment by muhdiboy** [comment]
Yes, this is to allow older builds or other Java dependent applications to be able to run on the same server. Because Minecraft is expecting the newest Java JDK 16, many servers I manage or know of will have dependency issues.

**Comment by MrGraversen** [comment]
Yes. The JDK 16 update has proven to cause trouble when using e.g. `crontab` to schedule `mcserver monitor` commands, etc.
